app/controllers/application_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
+  $studio_due = Time.parse("2016/01/10 23:59")
```

Comment:
I would recommend put this global setting to the initializer folder, see this link: http://stackoverflow.com/questions/1450285/how-to-define-custom-configuration-variables-in-rails